scripts/hail_batch/snp_chip_generate_pca/snp_chip_generate_pca.py

```diff
@@ -0,0 +1,47 @@
+"""
+Generate PCA on combined TOB-WGS/SNP-chip data.
+"""
+
+import click
+import hail as hl
+import pandas as pd
+
+TOB_WGS = 'gs://cpg-tob-wgs-main/mt/v2-raw.mt/'
+SNP_CHIP = 'gs://cpg-tob-wgs-test/snpchip/v1/snpchip_grch38.mt/'
+
+
+@click.command()
+@click.option('--output', help='GCS output path', required=True)
+def query(output):  # pylint: disable=too-many-locals
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    tob_wgs = hl.read_matrix_table(TOB_WGS).key_rows_by('locus', 'alleles')
+    snp_chip = hl.read_matrix_table(SNP_CHIP).key_rows_by('locus', 'alleles')
+
+    # filter to loci that are contained in snp-chip data after densifying
+    tob_wgs = hl.experimental.densify(tob_wgs)
+    tob_wgs = tob_wgs.select_entries(tob_wgs.GT).select_cols()
+    snp_chip = snp_chip.select_entries(snp_chip.GT).select_cols()
```

Comment:
```suggestion
    tob_wgs = tob_wgs.select_cols().select_entries(tob_wgs.GT)
    snp_chip = snp_chip.select_cols().select_entries(snp_chip.GT)
```

Reply:
Just out of curiosity, does the ordering matter here? Or is it about readability?